gnd/src/main/java/com/google/android/gnd/persistence/remote/RemoteDataStore.java

```diff
@@ -57,15 +57,15 @@ public interface RemoteDataStore {
   Maybe<TermsOfService> loadTermsOfService();
 
   /**
-   * Returns all features in the specified survey, then continues to emit any remote updates to the
-   * set of features in the survey until all subscribers have been disposed.
+   * Returns all lois in the specified survey, then continues to emit any remote updates to the set
```

Comment:
This is an acronym, so in comments this should be capitalized as "LOI" or "LOIs".

Reply:
Done